src/ui/components/Card/Card.test.tsx

```diff
@@ -138,10 +140,11 @@ describe('Card', () => {
     ({ gameState }) => {
       const send = mockSend()
       vi.spyOn(useGameStateModule, 'useGameRules').mockReturnValueOnce({
-        eventCardsThatCanBePlayed: 1,
         gameState,
-        game: stubGame(),
-        selectedWaterCardInHandIdx: defaultSelectedWaterCardInHandIdx,
+        game: stubGame({
+          selectedWaterCardInHandIdx: defaultSelectedWaterCardInHandIdx,
+          winner: null,
+        }),
         winner: null,
```

Comment:
It seems that `winner` should only be within `game` here, so this line can probably be removed.